File: Go/Slices/add-to-slice.go
package main

import (
	"fmt"
)

func main() {
	// Create a slice of strings
	var randomSlice []string
	// Append some values to the slice
	randomSlice = appendToSlice(randomSlice, "Hello")
}

// Append some string to a slice and than return the slice.
func appendToSlice(slice []string, content string) []string {
	// Append the content to the slice
	slice = append(slice, content)
	// Return the slice
	return slice
}

File: TypeScript/Slice/add-to-slice.ts
function main(): void {
    // Create a slice.
    let randomData: string[] = ["a", "b", "c"]
    // Add an element to the slice.
    randomData = appendToSlice(randomData, "d")
    // Print the slice.
    console.log(randomData)
}

main()

// Add an element to the slice and return the new slice.
function appendToSlice(slice: string[], element: string): string[] {
    return slice.concat(element)
}